CONTRIBUTING.md

```diff
@@ -1,4 +1,79 @@
-# Developer's Certificate of Origin 1.1
+# Contributing to Undici
+
+* [Guides](#guides)
+  * [Update `llhttp`](#update-llhttp)
+* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin)
+  * [Moderation Policy](#moderation-policy)
+
+<a id="guides"></a>
+## Guides
+
+<a id="update-llhttp"></a>
+### Update `llhttp`
+
+The HTTP parser used by `undici` is a WebAssembly build of [`llhttp`](https://github.com/nodejs/llhttp).
+
+While the project itself provides a way to compile targeting WebAssembly, at the moment we embed the sources
+directly and compile the module in `undici`.
+
+The `deps/llhttp/include` folder contains the C header files, while the `deps/llhttp/src` folder contains
+the C source files needed to compile the module.
+
+The `lib/llhttp` folder contains the `.js` transpiled assets required to implement a parser.
+
+The following are the steps required to perform an update.
+
+#### Clone the [llhttp](https://github.com/nodejs/llhttp) project
+
+```bash
+git clone git@github.com:nodejs/llhttp.git
+
+cd llhttp
+```
+#### Checkout a `llhttp` release
+
+```bash
+git checkout <tag>
+```
+
+#### Install the `llhttp` dependencies
+
+```bash
+npm i
+```
+
+#### Run the wasm build script
+
+```bash
+npm run build-wasm
+```
+
+#### Copy the sources to `undici`
```

Comment:
All these steps could be condensed to:

```bash
cp build/llhttp/wasm/* <path-to-undici>/lib/llhttp
```

If we remove the embedding and use the `llhttp` project build. Just wanted to say that's an option if this is too clumsy.

Reply:
This guide should be revisited if/when we have to modify `lhttp` sources for whatever reason.